### PR TITLE
ba/TSK-164-initial-commit-for-search-service

### PR DESCRIPTION
**Pull request by bugraaydin**
https://plentific.atlassian.net/browse/TSK-164

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+.pyre/
+
+# pytype static type analyzer
+.pytype/
+
+# Cython debug symbols
+cython_debug/
+
+
+.DS_Store
```
Comment (eemrebasar) [inline .gitignore:142]: should we push .gitignore?

**Review by LenardMarcin — APPROVED** [review]
looks good, thank you :+1:

File: .gitignore (hunk 1 above)
Comment (bugraaydin) [reply .gitignore:142]: @eemrebasar yep

File: .python-version [hunk 2]
```diff
@@ -0,0 +1 @@
+3.9.0
```
Comment (bugraaydin) [inline .python-version:1]: @eemrebasar push this as well btw

**Comment by bugraaydin** [comment]
note to myself: forgot the requirements file

**Pull request merged**